main/src/library/String.flix

```diff
@@ -1346,6 +1346,12 @@ mod String {
         else
             Some(dropRight(length(substr.substr), s))
 
+    ///
+    /// Unwrap the string, replacing every newline with a space
+    ///
+    pub def unwrap(s: String): String =
+        replaceChar({from = '\n'}, {to = ' '}, s)
```

Comment:
Should we take into account Windows-style and old Mac-style newlines? (CR LF and CR)